src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -132,4 +141,107 @@ public CommandBase turnDegrees(double degrees, double speed) {
         cmd.setName("Turn Degrees");
         return cmd;
     }
+
+    public CommandBase slowTurn(boolean direction) {
+        CommandBase cmd = new RunCommand(() -> {
+            if (direction == true) {
+                driveTrain.arcadeDrive(0, 0.4);
+            } else if (direction == false) {
+                driveTrain.arcadeDrive(0, -0.4);
+
+            }
+        }, this);
+        cmd.setName("turing");
+        return cmd;
+    }
+
+    public CommandBase visionAlignDegrees() {
+        CommandBase cmd = new CommandBase() {
+
+            @Override
+            public void initialize() {
+                gyro.reset();
+                pid.setSetpoint((SmartDashboard.getNumber("Angle Offset", 0)));
+                pid.setTolerance(1.5);
+                pid.setPID(0.025, 0.015, 0);
+
+            }
+
+            @Override
+            public void execute() {
+                if (i % 20 == 0) {
```

Comment:
I is a static variable in the subsystem. Can't it be an instance variable in this command?

Reply:
Stupid Java question, Can you make instance variables inside anonymous classes? Or does this need to become a full class implementation?

Reply:
Instance variables are fine - we use them extensively in chopshoplib